c/embeddedjs.c

```diff
@@ -0,0 +1,710 @@
+
+
+/*
```

Comment:
Does this require QuickJS? If so, should we include it as a dependency in zowe-common-c? Attribution might also be required because of the license.

Reply:
Sean has a list of zowe dependencies that shows up somewhere for license compliance.